modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-feature-search/ddm-template.ftl

```diff
@@ -0,0 +1,399 @@
+											<#assign url = nestedFieldData.contentFieldValue.data />
+										</#if>
+									<#else>
+										<#assign urlTitle = "" />
+									</#if>
+								</#list>
+
+								<#if urlTitle?has_content && validator.isNotNull(urlTitle)>
+									<div class="d-flex mb-2">
+										<a class="link-container" href="${url}" target="_blank">
+											<@clay["icon"] symbol="link" />
+
+											<span>
+												${urlTitle}
+											</span>
+										</a>
+									</div>
+								</#if>
+							</#list>
+						</div>
+
+						<div class="ml-auto mr-5 mt-3">
+							<#list restArticle.taxonomyCategoryBriefs as taxonomyCategoryBrief>
+								<#assign taxonomyVocabularyName = taxonomyCategoryBrief.embeddedTaxonomyCategory.parentTaxonomyVocabulary.name />
+
+								<#if taxonomyVocabularyName == "Release Status Previous">
+									<span class="font-weight-normal label label-previous-status label-secondary label-tonal-secondary px-2 text-paragraph-sm">
+										${taxonomyCategoryBrief.taxonomyCategoryName}
+
+										<span class="pl-1">
+											<@clay["icon"] symbol="info-circle" />
+										</span>
+									</span>
+									<span class="mr-1">
+										<@clay["icon"] symbol="order-arrow-right" />
+									</span>
+								</#if>
+							</#list>
+
+							<#list restArticle.taxonomyCategoryBriefs as taxonomyCategoryBrief>
+								<#assign taxonomyVocabularyName = taxonomyCategoryBrief.embeddedTaxonomyCategory.parentTaxonomyVocabulary.name />
+
+								<#if taxonomyVocabularyName == "Feature Availability">
+									<span class="font-weight-normal label label-success label-tonal-success mb-3 px-2 text-paragraph-sm">
+										${taxonomyCategoryBrief.taxonomyCategoryName}
+
+										<span class="pl-1">
+											<div class="clay-tooltip-top fade tooltip" id="tooltipId-${searchEntry.getClassPK()}" role="tooltip">
+												<div class="arrow"></div>
+
+												<div class="tooltip-inner">
+													<a href="https://learn.liferay.com/en/w/dxp/system-administration/configuring-liferay/feature-flags#beta-features-and-feature-flags" target="_blank">
+														${languageUtil.get(locale, "learn-about-the-feature-availability-statuses", "Learn about the feature availability statuses.")}
+													</a>
+												</div>
+											</div>
+
+											<button class="btn-tooltip-availability tooltip-button" data-tooltip-id="tooltipId-${searchEntry.getClassPK()}" type="button">
+												<@clay["icon"] symbol="info-circle" />
+											</button>
+										</span>
+									</span>
+								</#if>
+							</#list>
+						</div>
+					</div>
+				</div>
+
+				<hr class="solid">
+			</#if>
+		</#list>
+	<#else>
+		<div class="empty-container">
+			<div class="d-flex empty-result-message flex-column">
+				<div class="icon-container">
+					<div class="search-icon">
+						<@clay["icon"] symbol="search-experiences" />
+					</div>
+				</div>
+
+				<h1 class="empty-message-title text-center">
```

Comment:
h1s are typically meant to be the main heading for the page. If we want to borrow the styles/size of an h1 usually there is a class we can add to another element (ie something like .display-size-1 that you add to an h2 or span)